BlazarService/src/main/java/com/hubspot/blazar/config/BlazarSentryConfiguration.java

```diff
@@ -0,0 +1,99 @@
+package com.hubspot.blazar.config;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.getsentry.raven.logback.SentryAppender;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.Appender;
+import io.dropwizard.logging.AbstractAppenderFactory;
+import io.dropwizard.logging.async.AsyncAppenderFactory;
+import io.dropwizard.logging.filter.LevelFilterFactory;
+import io.dropwizard.logging.layout.LayoutFactory;
+
+/**
+ * This class configures sentry via a Logback appender the resource
+ * file META-INF/services/io.dropwizard.logging.AppenderFactory is responsible
+ * for telling Dropwizard's logging mechanics to create an appender using this class
+ * provided that the logging configuration of type "sentry" is available.
+ *
+ * <p>
+ * sentryDsn: string (Data Source Name) is the secret Sentry gives you to report exceptions with.
+ * sentryRelease: string the version of blazar you are deploying. Sentry will tag exceptions using this release value
+ *
+ * This appender is only used if you configure the logging appender in the dropwizard config for sentry as follows:
+ * {@Code
+ *  logging:
+ *    appenders:
+ *      - type: sentry
+ *        sentryDsn: http://some_Sentry_provided_DSN
+ *        sentryRelease: "PROD"
+ * }
+ *
+ */
+@JsonTypeName("sentry")
+public class BlazarSentryConfiguration extends AbstractAppenderFactory<ILoggingEvent> {
```

Comment:
Seems that this class serves as both the appender factory and special dropwizard config class but better name it `SentryAppenderFactory`. It would more easy to understand its main purpose.